TLM/TLM/Util/DetailLogger.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using CSUtil.Commons;
+
+namespace TrafficManager.Util {
+    internal static class DetailLogger {
```

Comment:
we can have a philosophical debate about whether this should be in TLM  or in CSUtils.commons.

Reply:
I'm following the principle of minimizing the scope until potential outside use cases are identified and better understood.